File: 백준/Bronze/32905. RACI/RACI.py
import sys
input=lambda:sys.stdin.readline().rstrip()
n,m=map(int,input().split())
flag=True
for _ in range(n):
    arr=input().split()
    if arr.count('A')!=1:
        flag=False
        break
print("Yes" if flag else "No")

File: 백준/Bronze/32905. RACI/RACI.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int n, m, cnt;
    bool flag = true;
    cin >> n >> m;
    vector<vector<char>> arr(n, vector<char>(m, 0));
    for (int i = 0; i < n; i++) {
        cnt = 0;
        for (int j = 0; j < m; j++) {
            cin >> arr[i][j];
            if (arr[i][j] == 'A')
                cnt += 1;
        }
        if (cnt != 1) {
            flag = false;
            break;
        }
    }
    cout << (flag ? "Yes" : "No");
    return 0;
}